### PR TITLE
Migrate `multiply` plugin to `builtins`

### PR DESCRIPTION
**Pull request by jmcook1186**
### Types of changes

- [x] Enhancement (project structure, spelling, grammar, formatting)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### A description of the changes proposed in the Pull Request

Migrates `Multiply` plugin into IF `builtins`.
Addresses https://github.com/Green-Software-Foundation/if/issues/679

### CODE REVIEW

File: src/builtins/multiply/README.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+```pseudocode
+output = input0 * input1 * input2 ... inputN
+```
+
+## Implementation
+
+To run the plugin, you must first create an instance of `Multiply`. Then, you can call `execute()`.
+
+```typescript
+import {Multiply} from '@grnsft/if-plugins';
```
Comment (jmcook1186) [inline src/builtins/multiply/README.md:37]:
```suggestion
import {Multiply} from 'builtins';
```

File: src/builtins/multiply/README.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+name: multiply-demo
+description:
+tags:
+initialize:
+  outputs:
+    - yaml
+  plugins:
+    multiply:
+      method: Multiply
+      path: '@grnsft/if-plugins'
```
Comment (jmcook1186) [inline src/builtins/multiply/README.md:69]:
```suggestion
      path: 'builtins'
```

File: src/builtins/multiply/README.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+          duration: 3600
+          cpu/energy: 0.001
+          network/energy: 0.001
+```
+
+You can run this example by saving it as `./examples/manifests/test/multiply.yml` and executing the following command from the project root:
+
+```sh
+npm i -g @grnsft/if
+npm i -g @grnsft/if-plugins
```
Comment (narekhovhannisyan) [inline src/builtins/multiply/README.md:91]: dont have to install plugins

Comment (jmcook1186) [reply src/builtins/multiply/README.md:91]:
```suggestion
```

**Review by manushak — APPROVED** [review]

File: src/builtins/multiply/README.md [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+name: multiply-demo
+description:
+tags:
+initialize:
+  outputs:
+    - yaml
+  plugins:
+    multiply:
+      method: Multiply
+      path: 'builtins'
```
Comment (manushak) [inline src/builtins/multiply/README.md:69]: should be `builtin`

Comment (jmcook1186) [reply src/builtins/multiply/README.md:69]:
```suggestion
      path: 'builtin'
```

File: src/builtins/multiply/README.md [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+```pseudocode
+output = input0 * input1 * input2 ... inputN
+```
+
+## Implementation
+
+To run the plugin, you must first create an instance of `Multiply`. Then, you can call `execute()`.
+
+```typescript
+import {Multiply} from 'builtins';
```
Comment (narekhovhannisyan) [inline src/builtins/multiply/README.md:37]: do we need this part for builtin plugins?

**Review by narekhovhannisyan — APPROVED** [review]

File: src/builtins/multiply/README.md (hunk 5 above)
Comment (jmcook1186) [reply src/builtins/multiply/README.md:37]: I think so... ?

**Pull request merged**